source/simulator/assembly.cc

```diff
@@ -1192,7 +1274,17 @@ namespace aspect
   template <int dim>
   void Simulator<dim>::assemble_stokes_system ()
   {
-    computing_timer.enter_section ("   Assemble Stokes system");
+    if (!assemble_newton_stokes_system)
+      computing_timer.enter_section ("   Assemble Stokes system");
+    else if (assemble_newton_stokes_matrix)
+      {
+        if (parameters.newton_theta == 0)
+          computing_timer.enter_section ("   Assemble Stokes system picard");
```

Comment:
Is this not the same as the "Assemble Stokes system" output above? I would prefer if we keep the same output for the same operation